nxc/protocols/smb.py

```diff
@@ -879,8 +880,10 @@ def shares(self):
             write = False
             write_dir = False
             write_file = False
+            pwd = ntpath.join("\\", "*")
```

Comment:
:thinking:

Reply:
OH, this is the code I use to debug before, will remove it